eng/common-tests/spelling/check-spelling-in-changed-files.tests.ps1

```diff
@@ -0,0 +1,196 @@
+Describe 'Spell checking' -Tag 'UnitTest' {
+    BeforeAll {
+        $workingDirectory = Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+
+        Write-Host "Create test temp dir: $workingDirectory"
+        New-Item -ItemType Directory -Force -Path $workingDirectory | Out-Null
+
+        Push-Location $workingDirectory | Out-Null
+        git init
+
+        # Set up git user information for this specific repo if no user
+        # information is set globally. Windows DevOps machines do not have a
+        # global git identity set.
+        if (!(git config --get user.email)) {
+            git config user.email "test@contoso.com"
+        }
+        if (!(git config --get user.name)) {
+            git config user.name "Test User"
+        }
+
+        New-Item -ItemType Directory -Force -Path "./excluded"
+        New-Item -ItemType Directory -Force -Path "./included"
+        New-Item -ItemType Directory -Force -Path "./.vscode"
+
+        "Placeholder" > "./excluded/placeholder.txt"
+        "Placeholder" > "./included/placeholder.txt"
+
+        $configJsonContent = @"
+{
+    "version": "0.2",
+    "language": "en",
+    "ignorePaths": [
+        ".vscode/cspell.json",
+        "excluded/**"
+    ]
+}
+"@
+        $configJsonContent > "./.vscode/cspell.json"
+
+        git add -A
+        git commit -m "Init"
+
+        # This forces normal Write-Error behavior (as seen on desktop) for
+        # errors encountered during testing of
+        # check-spelling-in-changed-files.ps1. Setting this keeps errors out of
+        # the DevOps logs for successful tests. Any errors produced by these
+        # tests are expected or will result in test failures.
+        $OriginalTeamProjectId = $env:SYSTEM_TEAMPROJECTID
+        $env:SYSTEM_TEAMPROJECTID = $null
```

Comment:
This is needed to prevent DevOps from showing a bunch of fake error messages produced by the testing.